index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio mobile first</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
- [ ] please ensure that the toolbar section is inside a `header `tag not a div.[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info